lnrpc/rpc.proto

```diff
@@ -2242,6 +2242,11 @@ message Invoice {
 
     /// List of HTLCs paying to this invoice [EXPERIMENTAL].
     repeated InvoiceHTLC htlcs = 22 [json_name = "htlcs"];
+
+    /**
+    The list of private channel ids that will be used as hop hints.
```

Comment:
Add a comment that hop hints will be strictly limited to these channels if they are specified?